web/assets/main.scss

```diff
@@ -23,3 +32,65 @@ $moj-page-width: 1220px;
     margin-left: 50%;
     transform: translateX(-50%);
 }
+
+.app-c-option-select {
+      position: relative;
+      padding: 0 0 10px;
+      margin-bottom: 10px;
+      border-bottom: 1px solid $govuk-border-colour;
+  }
```

Comment:
Probably worth spinning off all the `app-c-option-select*` classes into a separate SCSS file so it's a bit easier to manage